src/lib/consts.ts

```diff
@@ -113,4 +113,5 @@ export enum CommandExitCodes {
 	InvalidActorJson = 5,
 
 	NotImplemented = 255,
+	NotFound = 404,
```

Comment:
The exit code cannot be larger than 255 :neutral_face: Please pick something else.